packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -354,6 +354,8 @@ export class GroupedListSection extends React.Component<IGroupedListSectionProps
 
     const nestingDepth = subGroup.level ? subGroup.level + 1 : groupNestingDepth;
 
+    const groupHeaderProps = { ...headerProps, groupLevel: nestingDepth };
```

Comment:
Unfortunately, changing `groupHeaderProps` to a `const` will not resolve the re-render of `GroupedListSection` issue mentioned in https://github.com/OfficeDev/office-ui-fabric-react/pull/11494#discussion_r359576553. With this change, `groupHeaderProps` is being assigned to a new object reference in memory. When `GroupedListSection` does a referential equality check in `shouldComponentUpdate` e.g. `oldProp === newProp` it will evaluate to `false` always, causing a re-render.

To resolve, either the object needs to not be recreated in-line each render pass when `groupLevel` does not change or special handling needs to be added to `GroupedListSection` given the current prop signature.

Reply:
turns out this wasn't necessary afterall. Not sure why I even added it.